site/src/atomics/spacing.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+These classes can be used to add spacing between elements.
+
+## Available sizes
+
+```Text
+`-none`
+`-xs`
+`-s`
+`-m`
+`-l`
+`-xl`
+`-xxl`
+`-xxxl`
+```
+
+## Margin and padding helpers
+
+For each of the spacing sizes, there are margin and padding helpers available:
+<br/><br/>
+
+| Size | Value         | Margin       | Padding       |
+| :--- | :------------ | :----------- | :------------ |
+| None | 0             | .margin-none | .padding-none |
+| XS   | 1rem / 16px   | .margin-xs   | .padding-xs   |
+| S    | 1.5rem / 24px | .margin-s    | .padding-s    |
+| M    | 2rem / 32px   | .margin-m    | .padding-m    |
+| L    | 3rem / 48px   | .margin-l    | .padding-l    |
+| XL   | 4rem / 64px   | .margin-xl   | .padding-xl   |
+| XXL  | 6rem / 96px   | .margin-xxl  | .padding-xxl  |
+| XXXL | 8rem / 128px  | .margin-xxxl | .padding-xxxl |
+
+### Margin Example
+
+```html
+<div class="border">
+	<div class="margin-xs border">Margin</div>
```

Comment:
Should the text in element should be the size?

Reply:
updated